src/Sarif/Core/MultiformatMessageString.cs

```diff
@@ -7,6 +7,6 @@ public partial class MultiformatMessageString
     {
         public bool ShouldSerializeMarkdown() { return !string.IsNullOrWhiteSpace(this.Markdown); }
 
-        public bool ShouldSerializeText() { return !string.IsNullOrWhiteSpace(this.Text); }
+        public bool ShouldSerializeText() { return true; }
```

Comment:
>true [](start = 51, length = 4)

True is a default for all serialization, and so if this change were correct, we should be able to remove it entirely. I think we added this special logic, though, to handle the case where users populated text with a non-null but empty string.

Reply:
See my comment to Larry on #1740, I'm not convinced we have the right fix here.

---
In reply to: [432989835](https://github.com/microsoft/sarif-sdk/pull/1885#discussion_r432989835) [](ancestors = 432989835)